### DISCUSSION TITLE
Waiting for first pageview

### DISCUSSION THREAD

**Discussion by stccorp** [plausible/analytics#990] [Self-Hosted Support] [answered]
Hello,

I am using ubuntu 20. I install docker and plausible. I got the email with the code. I install nginx, and set it up, forwarding to 8000 port as suggested here(https://theselfhostingblog.com/posts/completely-self-hosting-plausible-io-a-privacy-friendly-alternative-to-google-analytics/#step-2-installing-plausible-io).  I also install certbot for https.  I also setup the domain.  So when  I visit
https://MY DOMAIN HERE/js/plausible.js, I can see the javascript file.

BUT, when I log in and try to see the stats, all I get is "Waiting for first pageview".

I am not sure where to go from here.

I am getting plausible.js net::ERR_BLOCKED_BY_CLIENT

**Comment by maciekmm** [comment, marked as answer]
It looks like you have an ad blocker. You can get your first page view by turning it off.
EasyList, which is used by a significant number of ad blocking software contains a rule blocking `plausible.js`.
To circumvent this, you might want to add a rewrite rule to your nginx such as:
```
    location /js/plaus.js {
        rewrite /js/plaus.js(.*) /js/plausible.js$1 break;
        proxy_pass http://plausible:8000;
        proxy_redirect off;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header Host $host;
    }
```
And then change your html to reference `plaus.js` instead of `plausible.js`

**Reply by stccorp** [reply]
Thank you so much. That worked BUT....  Now the javascript is not blocked,  but the content is not executed because mixed content.  Inside plausible.js I can see my domain as http://MY DOMAIN. 

Where can I change the configuration so that it contains https://MY_DOMAIN inside plausible.js

**Reply by stccorp** [reply]
And the other error I am getting now is Status CORS error, Cross-Origin Resource Sharing error:  Missing Allow Origin Header
![brave_NyzgRZGMYK](https://user-images.githubusercontent.com/8180135/117081913-ea7add00-ad0e-11eb-9ca1-8a49fc0c218e.png)

**Reply by maciekmm** [reply]
Are you getting the CORS error from a domain you have added via plausible dashboard?
